src/store/middlewares/peers.js

```diff
@@ -10,7 +11,10 @@ const peersMiddleware = store => next => (action) => {
 
   switch (action.type) {
     case actionTypes.storeCreated:
-      store.dispatch(activePeerSet({ network }));
+      // It changes network back to mainnet when we want to autologin for explorer account
+      if (!shouldAutoLogIn) {
```

Comment:
`shouldAutoLogIn` us a function that should be used like this:
https://github.com/LiskHQ/lisk-hub/blob/9a74df7a8f9a3a333d307efc2b3b2115f6cbe1e8/src/store/middlewares/account.js#L124

Currently, the condition is never met.